apps/native-component-list/src/screens/AV/Player.tsx

```diff
@@ -51,6 +52,7 @@ interface Props {
   shouldCorrectPitch: boolean;
   isPlaying: boolean;
   isMuted: boolean;
+  metadata: AVMetadata;
```

Comment:
We've got CI error on this one: https://github.com/expo/expo/pull/14134/checks?check_run_id=3505911371#step:7:6
We need to make it optional or adjust the VideoPlayer component that is also based on this class.
```suggestion
  metadata?: AVMetadata;
```